src/assets/_project/_blocks/layout/section-nav/step-nav.js

```diff
@@ -1,14 +1,14 @@
 import breakpoints from '../../utils/breakpoints.js';
 let stepNav = {
   config: {
-    $guideSubNav: $('#qg-section-nav .guide-sub-nav'),
-    $qgSectionNav: $('#qg-section-nav'),
-    $qgSectionNavListItems: $('#qg-section-nav .guide-sub-nav li'),
+    $guideSubNav: $('.qg-section-nav .guide-sub-nav'),
```

Comment:
so from div id to class id?

Reply:
Yes, new section nav theme is implemented using .qg-section-nav class so that we don't change the existing theme in applications like Seniors etc which are using id.